Makefile

```diff
@@ -117,6 +117,18 @@ clean:
 coverage:
 	@scripts/coverage.sh
 
+.PHONY: dind
+dind: docker-build
+dind:
+	@scripts/portforward.sh start
+	@scripts/dind.sh
+	@scripts/import-docker-image.sh
+
+.PHONY: e2e
+e2e: dind
+e2e:
+	go test -v ./e2e --cluster-url http://localhost:8080 --helm-bin ../bin/helm
```

Comment:
The cluster URL should probably use variables (set with this default value) so you can override them if testing against another cluster of your choice